<launch>
  <!--<node pkg="tf" type="static_transform_publisher" name="world_to_cam"
    args="1 3 2 0.2 0 0 map bottom_cam 100" />-->
  <node ns="ardrone" name="bottom" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="video_device" value="/dev/video0" />
    <param name="image_width" value="640" />
    <param name="image_height" value="480" />
    <param name="pixel_format" value="yuyv" />
    <param name="camera_frame_id" value="bottom_cam" />
    <param name="io_method" value="userptr"/>
    <param name="framerate" value="25"/>
    <param name="camera_info_url" value="package://iarc_vision/calibration/bottom_facing_camera.yaml"/>
  </node>
  <!--<node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
    <remap from="image" to="/ardrone/bottom/image_raw"/>
    <param name="autosize" value="true" />
  </node>-->
</launch>
